From 53a858e2bf033d229df781b202d04f79e740c9d7 Mon Sep 17 00:00:00 2001
From: Prasanth Jayachandran <prasanthj@apache.org>
Date: Thu, 18 Aug 2016 18:33:14 -0700
Subject: [PATCH 1020/1164] CDH-48309: HIVE-14559: Remove setting
 hive.execution.engine in qfiles (Prasanth
 Jayachandran reviewed by Siddharth Seth)

(cherry picked from commit 1dc02c1342383841e08bb46948b3d8bfcdc4b99d)

Change-Id: Ia47e2fb67f662dd7d7f8ce547e6bec427e893352
---
 .../test/queries/clientpositive/decimal_skewjoin.q |    2 --
 .../test/queries/clientpositive/orc_mr_pathalias.q |    1 -
 .../clientpositive/smb_join_partition_key.q        |    3 +--
 ql/src/test/queries/clientpositive/tez_fsstat.q    |    1 -
 4 files changed, 1 insertion(+), 6 deletions(-)

diff --git a/ql/src/test/queries/clientpositive/decimal_skewjoin.q b/ql/src/test/queries/clientpositive/decimal_skewjoin.q
index cac0d89..11c665a 100644
--- a/ql/src/test/queries/clientpositive/decimal_skewjoin.q
+++ b/ql/src/test/queries/clientpositive/decimal_skewjoin.q
@@ -1,5 +1,3 @@
-
-set hive.execution.engine=mr;
 set hive.optimize.skewjoin=true;
 set hive.auto.convert.join=false;
 
diff --git a/ql/src/test/queries/clientpositive/orc_mr_pathalias.q b/ql/src/test/queries/clientpositive/orc_mr_pathalias.q
index d77d07a..b92bb9c 100644
--- a/ql/src/test/queries/clientpositive/orc_mr_pathalias.q
+++ b/ql/src/test/queries/clientpositive/orc_mr_pathalias.q
@@ -7,7 +7,6 @@ insert overwrite table test.test_orc_src select 1,2,3 from src limit 1;
 insert overwrite table test.test_orc_src2 select 1,2,4 from src limit 1;
 
 set hive.auto.convert.join = false;
-set hive.execution.engine = mr;
 
 select
     tb.c
diff --git a/ql/src/test/queries/clientpositive/smb_join_partition_key.q b/ql/src/test/queries/clientpositive/smb_join_partition_key.q
index 49e2d2f..a2d2720 100644
--- a/ql/src/test/queries/clientpositive/smb_join_partition_key.q
+++ b/ql/src/test/queries/clientpositive/smb_join_partition_key.q
@@ -1,4 +1,3 @@
-SET hive.execution.engine=mr; 
 SET hive.enforce.sortmergebucketmapjoin=false; 
 SET hive.auto.convert.sortmerge.join=true; 
 SET hive.optimize.bucketmapjoin = true; 
@@ -32,4 +31,4 @@ STORED AS ORC;
 
 INSERT OVERWRITE TABLE smb_table_part PARTITION (p1) SELECT key, value, 100 as p1 FROM data_table;
 
-SELECT s1.key, s2.p1 FROM smb_table s1 INNER JOIN smb_table_part s2 ON s1.key = s2.key ORDER BY s1.key;
\ No newline at end of file
+SELECT s1.key, s2.p1 FROM smb_table s1 INNER JOIN smb_table_part s2 ON s1.key = s2.key ORDER BY s1.key;
diff --git a/ql/src/test/queries/clientpositive/tez_fsstat.q b/ql/src/test/queries/clientpositive/tez_fsstat.q
index 7f2e28f..5f2cce2 100644
--- a/ql/src/test/queries/clientpositive/tez_fsstat.q
+++ b/ql/src/test/queries/clientpositive/tez_fsstat.q
@@ -1,4 +1,3 @@
-set hive.execution.engine=tez;
 CREATE TABLE tab_part (key int, value string) PARTITIONED BY(ds STRING) CLUSTERED BY (key) INTO 4 BUCKETS STORED AS TEXTFILE;
 CREATE TABLE t1 (key int, value string) partitioned by (ds string) CLUSTERED BY (key) INTO 4 BUCKETS STORED AS TEXTFILE;
 
-- 
1.7.9.5

